sale_automatic_workflow/sale_workflow_process_view.xml

```diff
@@ -26,6 +26,7 @@
                     <group string="Workflow Options">
                         <field name="validate_order"/>
                         <field name="validate_picking"/>
+                        <field name="ship_only_available"/>
```

Comment:
If I am right, we must have the option "validate_picking" in order to enjoy the option "ship_only_available", if it's the case, then may we can hide the option "ship_only_available" and show it only it "validate_picking" is tic, what do you think?